setup.py

```diff
@@ -13,9 +13,16 @@
 # governing permissions and limitations under the License.
 
 from setuptools import setup, find_packages
+import os
+
+v = open('haas/version.py')
+exec(v.read())
```

Comment:
Two things:

1. We should be careful about how we're locating the file; some of the setup.py operations run things from other directories, so using a relative path could be perilous. See also #316 (which I *thought* was fixed...). I don't fully understand what needs to happen for this to be safe.
2. is the `exec` necessary? Is there a reason we couldn't just import it?

@gsilvis, pulling this from one place is going to necessarily involve a bit of trickery, can you weigh in on the right way to do this?